py/packages/corpora_ai/split.py

```diff
@@ -0,0 +1,41 @@
+import os
+from typing import Union
+
+from langchain_text_splitters import (
+    PythonCodeTextSplitter,
```

Comment:
TODO: this one is pretty meh ... should work harder to split at good places more than try to meet the Character specification - I'd rather have variable length with natural breaks than hard length with non-sense splits.